CHANGES.md

```diff
@@ -32,6 +32,13 @@ To be released.
  -  Added `IActionContext.NewGuId()` method.  [[#371], [#439]]
  -  `Address(byte[])` became to throw `ArgumentNullException`
     instead of `NullReferenceException`.  [[#443]]
+ -  `RawTransaction.Signer` became to `ImmutableArray<byte>` type.  [[#172], [#441]]
+ -  `RawTransaction.PublicKey` became to `ImmutableArray<byte>` type.  [[#172], [#441]]
+ -  `RawTransaction.Signature` became to `ImmutableArray<byte>` type.  [[#172], [#441]]
+ -  `RawTransaction.UpdatedAddresses` became to
+    `ImmutableArray<ImmutableArray<byte>>` type.  [[#172], [#441]]
+ -  `RawTransaction.Actions` became to
+    `IEnumerable<IImmutableDictionary<string, object>>` type.  [[#172], [#441]]
```

Comment:
I believe these changelogs don't have to be recorded because `RawTransaction` is an `internal` struct.  You could bypass the changelog checker by adding `[changelog skip]` in your commit message (any commit in this pull request).

Reply:
I got it! I’ll bypass the changelog ci.